src/sentry/transactionHub.ts

```diff
@@ -0,0 +1,37 @@
+import * as Sentry from '@sentry/react-native'
+import { Transaction, TransactionContext } from '@sentry/types'
+
+// @ts-ignore
+export class TransactionHub {
+  private static instance: TransactionHub
+  transactions: Array<Transaction>
+
+  private constructor() {
+    this.transactions = []
+  }
+
+  public static getInstance(): TransactionHub {
+    if (!TransactionHub.instance) {
+      TransactionHub.instance = new TransactionHub()
+    }
+    return TransactionHub.instance
+  }
+
+  public startTransaction(transactionContext: TransactionContext) {
```

Comment:
this is not a fully thought out suggestion, so i'm wondering what you think about this:

since we're abstracting `Sentry.startTransaction`, i'm thinking we could make the input arguments of this method a little more friendly (instead of very bespoke Sentry arguments). it can be as simple as just having the input arguments spread out rather than one object, since that already helps with type prompts when writing new code. 

in ValoraAnalytics we have some guidance for what we can pass into the `track` method. i wonder if we should consider doing something like this for sentry transactions too, so that consumers don't need to know the exact string value of the transaction. like `SentryTransactionHub.startTransaction(Transactions.SendOrInvite, Spans.Send)`

of course this means more abstractions/types/boilerplate :/ but in return we get an easier to use method + less chance of a typo (e.g. finishing a transaction that doesn't exist, so that the transaction that started will never be sent to Sentry) + a place where we can see all the transactions we're sending to sentry

Reply:
Added in https://github.com/valora-inc/wallet/pull/2728/commits/76f4ef86a8fc1d84cc74f72680ed4db2d4f51f09!